README.md

```diff
@@ -116,6 +116,8 @@ If you get `nvm: command not found` after running the install script, one of the
 
   - You might need to restart your terminal instance or run `. ~/.nvm/nvm.sh`. Restarting your terminal/opening a new tab/window, or running the source command will load the command and the new configuration.
 
+  - The file `~/.nvm/nvm.sh` might not have execution rights. Try running `chmod a+x ~/.nvm/nvm.sh` 
```

Comment:
This is incorrect; this file is not executable and must never _be_ executed - it should be sourced.

Can you help me understand why you think this is helpful?